### PR TITLE
#15805-expm1-torch-frontend

### LINKED ISSUE #15805: expm1

### PR DESCRIPTION
**Pull request by selvaraj-sembulingam**
Close #15805

### CODE REVIEW

**Comment by aparajith21** [comment]
add_frontend_checklist

File: ivy/functional/frontends/torch/tensor.py [hunk 1]
```diff
@@ -1048,6 +1048,9 @@ def count_nonzero(self, dim):
     def exp(self):
         return torch_frontend.exp(self)
 
+    def expm1(self):
```
Comment (aparajith21) [inline ivy/functional/frontends/torch/tensor.py:1051]: Hi! Thanks for contributing, but tests are failing on the CI and this is a change that needs to be made similar to other torch functions as bfloat16 and float16 are not supported for torch and this decorator needs to be added to mark it as unsupported. 

Feel free to request for another review once you've made this change.
```suggestion
@with_unsupported_dtypes({"2.0.1 and below": ("bfloat16", "float16")}, "torch")
    def expm1(self):
```

**Review by aparajith21 — CHANGES_REQUESTED** [review]
Hi, thanks for this PR. Just a few minor details to be fixed up and then we're ready to go for merging!

File: ivy/functional/frontends/torch/tensor.py (hunk 1 above)
Comment (selvaraj-sembulingam) [reply ivy/functional/frontends/torch/tensor.py:1052]: Added the unsupported types

**Comment by aparajith21** [comment]
Thank you for contributing! Merging this PR

**Pull request merged**